lib/database_admin/postgres.rb

```diff
@@ -0,0 +1,25 @@
+require 'sequel'
+
+module DatabaseAdministrator
+  class Postgres
+    def initialize
+      @migrator = Migrator.new
+    end
+
+    def fresh_database
+      database = existing_database
+
+      @migrator.destroy(database)
+      @migrator.migrate(database)
+
+      database
+    end
+
+    def existing_database
+      database = Sequel.connect('postgresql://admin:secretpassword@postgres/orders')
```

Comment:
Should this be an environment variable? You'll need this for your production/staging environment too and you'll want to avoid committing user/pass to a public repo